content/en/resources/mappings.md

```diff
@@ -8,16 +8,25 @@ author = ""
 weight = 3
 +++
 
-## OpenCRE
+## Direct Mappings
 
-The SAMM model pages on the website have links to OpenCRE in every stream. By linking SAMM to {{< external-link "https://www.opencre.org/" "OpenCRE">}}, we’ve made it easier for our users to find relevant and useful resources with every stream, as well as to see how SAMM aligns with other security standards such as NIST SSDF, ISO27K, PCI-DSS, OWASP ASVS, and NIST 800-53.
-
-Learn more in the <b>[OWASP SAMM now connects to OpenCRE](/blog/2023/09/20/owasp-samm-now-connects-to-opencre/)</b> blog post.
-
-## SAMM-NIST SSDF
+### Mapping between NIST SSDF and SAMM
 
 In collaboration with NIST, we created mappings based on the {{< external-link "https://csrc.nist.gov/projects/olir" "National Online Informative Reference (OLIR) Program">}}. An Informative Reference shows the relationships between the Reference Document elements (NIST SSDF Tasks) and a Focal Document element (OWASP SAMM Streams). This effectively helps users understand the characterization of the nature of each relationship.
 
 You can find the mapping in <b>{{< external-link "https://docs.google.com/spreadsheets/d/1AsIbEHK_csuYkUx8tSZvHBFlywYZ5wBejfJHi8AvnZM" "this spreadsheet">}}</b>. Note that we created the NIST SSDF to SAMM mapping. The reverse mapping is automatically generated and in the current version it is a crosswalk mapping.
 
-More on this in the <b>[Tackling App Security with SAMM-NIST SSDF Mapping](/blog/2023/02/06/samm-ssdf-mapping/)</b> blog post.
\ No newline at end of file
+More on this in the <b>[Tackling App Security with SAMM-NIST SSDF Mapping](/blog/2023/02/06/samm-ssdf-mapping/)</b> blog post.
+
+### Mapping between BSIMM 14 and SAMM
+The SAMM core team has created mappings between OWASP SAMM and BSIMM 14 standards. You can find the mapping in <b>{{< external-link "https://docs.google.com/spreadsheets/d/1WiQcn7wFzSM8xg78SqkIM1QF2C48jBCYi_N_6kOq174" "this spreadsheet">}}</b>.
+
+### Mapping between IEC-62443-4-1 and SAMM
+The SAMM core team has created mappings between OWASP SAMM and BSIMM 14 standards. You can find the mapping in <b>{{< external-link "https://docs.google.com/spreadsheets/d/1WiQcn7wFzSM8xg78SqkIM1QF2C48jBCYi_N_6kOq174" "this spreadsheet">}}</b>.
```

Comment:
Should the text here say "-- between OWASP SAMM and IEC-62443-4-1"? The text is the same as in section above now.